README.md

```diff
@@ -29,6 +30,42 @@ $ yarn add @shopify/preact-testing
 
 This library allows you to test Preact components with a focus on type safety and testing based on a component’s external API. In order to keep the API small and easy-to-use, it will generally track to only the latest minor release of Preact.
 
+### Basic
+
+A testcase using `@shopify/preact-testing` tends to have more or less the following structure:
+- [`mount`](#mount) your component with some props to get a wrapper
+- (optionally) mutate it using the wrapper's [`setProps`](#setProps) or by [`trigger`](#trigger)ing callbacks on elements within it
+- make assertions based on the wrapper (often using the custom [matchers](#matchers))
+
+#### Example
+
+This example uses [jest](https://jestjs.io/) as a test-runner.
+
+```tsx
+import {h} from 'preact';
+import {mount} from '@shopify/preact-testing';
+import ClickCounter from './ClickCounter';
+
+describe('<ClickCounter />', () => {
+  it('allows us to set props', () => {
+    const wrapper = mount(<Foo defaultCount={0} />);
+    
+    expect(wrapper.props().count).toBe(0);
+    expect(wrapper.text()).toBe('count: 0');
+    wrapper.setProps({defaultCount: 1});
+    expect(wrapper.props().count).toBe(1);
+    expect(wrapper.text()).toBe('count: 0');
```

Comment:
this last assertion is a bit confusing without seeing `Foo`, its not updated to 1 because you need to re-render it?

Reply:
it should be 1 I'm just bad